.travis.yml

```diff
@@ -1,12 +1,22 @@
 language: java
-sudo: false
+sudo: required
```

Comment:
Ah yes so it is the old and non-dockerized travis environment (hence already the higher boot time), but they do not intend shutting that down any time soon.